satellite/satellitedb/offers.go

```diff
@@ -118,24 +118,6 @@ func (db *offersDB) Create(ctx context.Context, o *rewards.NewOffer) (*rewards.O
 	return newOffer, offerErr.Wrap(tx.Commit())
 }
 
-// Redeem adds 1 to the amount of offers redeemed based on offer id
-func (db *offersDB) Redeem(ctx context.Context, oID int, isDefault bool) error {
-	if isDefault {
-		return nil
-	}
-
-	statement := db.db.Rebind(
-		`UPDATE offers SET num_redeemed = num_redeemed + 1 where id = ? AND status = ? AND num_redeemed < redeemable_cap`,
```

Comment:
It looks like this change makes `num_redeemed` an unused column. Is it worth removing it from the schema so people don't get confused and think that it has meaning?

Reply:
Do you think it would be better to do it in another PR? In order to remove this column, I will need to change the logic of getting current offer. That seems to make this PR too big

Reply:
I did it in another PR #2510